README_CONTAINER_IMAGE.md

```diff
@@ -38,4 +44,20 @@ Here is an example of how to run a containerized `openshift-ansible` playbook th
            -e PLAYBOOK_FILE=playbooks/certificate_expiry/default.yaml \
            openshift/openshift-ansible
 
-The [playbook2image examples](https://github.com/aweiteka/playbook2image/tree/master/examples) provide additional information on how to use an image built from it like this one.
+#### Scheduled job
```

Comment:
Please please please please please **do not change the extensions** in the `certificate_expiry` directory. Those files referenced in KB articles and online documentation. Changing those extensions will cause support cases to roll in as customers can not run the expiration checks as documented.